### PR TITLE
Displacement filter

### PR DESCRIPTION
**Pull request by amylizzle**
image

### CODE REVIEW

File: TestGame/icons/displace.dmi
Comment (wixoaGit) [inline TestGame/icons/displace.dmi:1]: This needs added to the [solution file](https://github.com/OpenDreamProject/OpenDream/blob/master/OpenDream.sln#L15) so it's visible within the IDE

**Review by wixoaGit — CHANGES_REQUESTED** [review]
The filter doesn't have the same affect as it does in BYOND
image
BYOND on the left

**Comment by amylizzle** [comment]
Close, but no cigar 
image

**Comment by amylizzle** [comment]
Done 
image

Minor differences are due to sampling algorithm and different `world.view` sizes I'm pretty sure. Sampling algo isn't exposed to content though, so we'll have to fix that in the upstream RT.

**Review by wixoaGit — APPROVED** [review]

**Pull request merged**